include/sound/sof/topology.h

```diff
@@ -218,21 +218,25 @@ struct sof_ipc_comp_reply {
  * Pipeline
  */
 
```

Comment:
Jo @tlauda,

Sorry for being late. One minor comment here. Please use more specific subject line. "Update" is just too generic.

For example, you can say ASoC: SOF: Align sof_ipc_pipe_new with FW code

Reply:
@dbaluta Thanks for the suggestion. I'll try to do better next time.